.circleci/config.yml

```diff
@@ -480,9 +562,6 @@ jobs:
           keys:
           - v3-front-dependencies-{{ checksum "package.json" }}
           - v3-front-dependencies-
-      # Copy an up-to-date yarn.lock file
```

Comment:
Why are we removing this?

Reply:
Because nothing has been saved to the workspace, hence there is nothing that needs to be restored/attached.

Reply:
I'm not sure what's generally going on, was just pointing it out because it seemed unrelated to the rest of the PR.

Anyway 👍

Reply:
You are right: it should have been cleaned when we removed the yarn lockfile generation step.